frontend/src/pages/Sessions/SessionsFeedV3/SessionQueryBuilder/components/QueryBuilder/QueryBuilder.tsx

```diff
@@ -1307,6 +1245,38 @@ enum QueryBuilderMode {
 	SEGMENT_UPDATE = 'SEGMENT_UPDATE',
 }
 
+const now = moment()
+const defaultMinDate = now.clone().subtract(90, 'days').toDate()
+
+const presetOptions = [
+	{
+		label: 'Last 15 minutes',
+		startDate: now.clone().subtract(15, 'minutes').toDate(),
+	},
+	{
+		label: 'Last 60 minutes',
+		startDate: now.clone().subtract(60, 'minutes').toDate(),
+	},
+	{
+		label: 'Last 4 hours',
+		startDate: now.clone().subtract(4, 'hours').toDate(),
+	},
+	{
+		label: 'Last 24 hours',
+		startDate: now.clone().subtract(24, 'hours').toDate(),
+	},
+	{
+		label: 'Last 7 days',
+		startDate: now.clone().subtract(7, 'days').toDate(),
+	},
+	{
+		label: 'Last 30 days',
+		startDate: now.clone().subtract(30, 'days').toDate(),
+	},
+]
```

Comment:
Could these presets be reused across errors and sessions? Maybe we could just update the `makeDefaultPresets` in `PreviousDateRangePicker`.

https://github.com/highlight/highlight/blob/014de82d90e12bbb1fa8230cebaddcc87bbf7228/packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx/#L37-L54

Reply:
Hey @ccschmitz thanks for the feedback, i've made some changes